### DISCUSSION TITLE
Solving Problem With Not Syncing.

### DISCUSSION THREAD

**Discussion by allzima1** [Chia-Network/chia-blockchain#8970] [Support]
**Describe the bug**
after installing new version, the synchorazation is not gonna work.

**To Reproduce**

Resolving: 

Guys i found it finally. please follow me step by step:

1. fisrt of all close the app and unistall it completly.
2. open this pc and type this location: C:\users\username

![Capture111](https://user-images.githubusercontent.com/93139645/138901424-30ad4587-6819-4ad2-b644-92e1899945e9.PNG)

3. Then take a back up of the .chia folder at new folder on your Desktop for the emergency situation.
4. Then rename the .chia ( for example : .chia-node)
5. Then you need to run the latest version blockchain as administrator.
6. After all check it out if it is syncing or not.
7. if not, Then repeat the following (unistall blockchain) go to the mentioned folder again and delete the extra folder and rename the folder for the second time ( for example : .chia-node11 )
8. The difference between this option is, you must delete db in the .chia folder.
9 .now there we go it is working.
10. Guys listen: if your go to the wallet and see the money and coin is gone, Don't worry, go to the back up and copy all files from the wallet folder and paste them to the main folder in the wallet folder and replace the all.

Please Follow the Rules Step By Step Go Head!

**Expected behavior**

**Screenshots**

**Desktop**

- OS: Windows
- OS Version Enterprise
- CPU: Intel i9 11900

**Additional context**
For addressing the issue follow above

**Comment by sargonas** [comment]
Moved this to discussions, since "issues" eventually get closed and this is useful to keep referencing. This way, you can link back to this is topic in the issue comments for future folks to help them!
